web/src/components/nav_bar/nav_bar.styles.js

```diff
@@ -41,6 +44,13 @@ const navBarStyles = {
   logoContainer: {
     flex: 1,
   },
+  appBar: {
+    position: "fixed",
+    backgroundColor: theme.palette.primary.background.light,
+    boxShadow:
+      "0px 2px 2px -1px rgba(0,0,0,0.05), 0px 4px 5px 0px rgba(0,0,0,0.02), 0px 1px 10px 0px rgba(0,0,0,0.05);",
```

Comment:
Should we also make this a const?

Reply:
Isn't being used anywhere else

Reply:
yet

Reply:
we can do it when we do use it elsewhere